source/documentation/deploying-an-app/multicontainer-app-deploy.md

```diff
@@ -87,6 +87,28 @@ You should be able to view the application in your browser at:
 
 It should behave in the same way as when you were running it locally via docker-compose.
 
+### k8s_additional_resources
+
+After you deployed and Interacting with multi-container application and would like to create Monitoring dashboard , Custom Alerts and Cronjobs for your application use this working examples:
+
+* **Grafana dashboard** - Follow the [guide][creating-dashboards] on how to create a Graphana dashboard, make corresponding changes to [monitoring-grafana-dashboard.yaml][dashboard-configmap] file and apply the configmap to your namespace in the kubernetes cluster, which will show your Grafana-Dashboard [here][Dashboard].     
+
+      ```
+      kubectl apply --filename monitoring-grafana-dashboard.yaml --namespace [your namespace]
+      ```
+
+* **Custom Alerts** - Follow the [guide][custom-alert] on how to create Custom Alerts, make corresponding changes to [prometheus-app-alert.yaml][alert-prometheusrule] file and apply the PrometheusRule to your namespace in the kubernetes cluster, this will create custom alerts for your application.
+      
+      ```
+      kubectl apply --filename prometheus-app-alert.yaml --namespace [your namespace]
+      ```
+
+* **Cronjobs** - Follow the [guide] [cron-jobs] on creating a Cronjob, make corresponding changes to [cronjob-ecr.yaml][cronjob-yaml] file and apply the Cronjob to your namespace in the kubernetes cluster, this runs a job periodically on a given schedule to delete untagged images in the ecr-repo, this will help to limit the count of Images in the ecr-repo.
```

Comment:
The space between [guide] and [cron-jobs] needs to be removed.

Reply:
Thanks @digitalronin, fixed it